androidtv/basetv/basetv.py

```diff
@@ -201,7 +202,7 @@ def _cmd_current_app_media_session_state(self):
         ):
             return constants.CMD_CURRENT_APP_MEDIA_SESSION_STATE_GOOGLE_TV
 
-        # Is this an Android 11 device?
+        # Is this an Android 11-14 device?
         if self.DEVICE_ENUM == constants.DeviceEnum.ANDROIDTV and self.device_properties.get("sw_version", "") == "11":
```

Comment:
The comment says "Android 11-14 device" but the code only checks 11.  I think you meant to modify the code.

Reply:
No, that's just the comment :(